app/controllers/v0/terms_of_use_agreements_controller.rb

```diff
@@ -31,6 +33,10 @@ def decline
       render_error(e.message)
```

Comment:
we can rescue the common name error on these lines as well, just do:
`ActiveRecord::RecordInvalid, TermsOfUse::Exceptions::CommonNameNotFoundError => e`

And then if you raise the error with a message, then it will render properly here